databricks_cli/jobs/cli.py

```diff
@@ -40,17 +40,49 @@
 @require_config
 @eat_exceptions
 def create_cli(json_file, json):
+    """
+    Creates a job in the Databricks Job Service.
+
+    The specification for the json option can be found
+    https://docs.databricks.com/api/latest/jobs.html#create
+    """
     json_cli_base(json_file, json, create_job)
 
 
 @click.command(context_settings=CONTEXT_SETTINGS)
 @click.option('--json-file', default=None,
-              help='File containing json to POST to /jobs/create.')
+              help='File containing json to POST to /jobs/reset.')
 @click.option('--json', default=None, help='Displays absolute paths.')
+@click.option('--job-id', required=True, help='The job_id to reset')
 @require_config
 @eat_exceptions
-def reset_cli(json_file, json):
-    json_cli_base(json_file, json, reset_job)
+def reset_cli(json_file, json, job_id):
+    """
+    Resets (edits) the definition of a job.
+
+    NOTE. The json parameter is not the same as what is normally posted
+    in the request body to the reset endpoint. Instead it is the object
```

Comment:
The first sentence is redundant. People need to think  what is normally posted in the request body to the reset endpoint. Can we just explain what is the argument it takes?